compiler/rustc_passes/src/check_const.rs

```diff
@@ -47,6 +47,8 @@ impl NonConstExpr {
 
             Self::Match(TryDesugar(_)) => &[sym::const_try],
 
+            Self::Match(Postfix) => &[sym::postfix_match],
+
             // All other expressions are allowed.
             Self::Loop(Loop | While) | Self::Match(Normal | FormatArgs) => &[],
```

Comment:
```suggestion
            Self::Loop(Loop | While) | Self::Match(Normal | Postfix | FormatArgs) => &[],
```
I don't think there's any difference between prefix and postfix matches with regards to const context.

Reply:
Ah I misunderstood what this was doing.